week-6/schedule.md

```diff
@@ -0,0 +1,57 @@
+| _13:00_ | _Lunch_                                                                                     |                                       |
+| 14:00   | [Cookies workshop](https://github.com/foundersandcoders/ws-cookies)                         | Cookie management, authentication     |
+| 16:00   | Tech for Better                                                                             |                                       |
+
+## Day 2
+
+| Time    | Activity                                                                                                                          | Learning outcomes                         |
+| ------- | --------------------------------------------------------------------------------------------------------------------------------- | ----------------------------------------- |
+| 09:45   | Intro reading                                                                                                                     |                                           |
+| 10:00   | [Design burst](https://docs.google.com/presentation/d/1f8ryxVIngpu4KMv8rr5domBiCGKfgC3r25TstZbnvcg/edit#slide=id.g26a95a14fb_0_0) | Typography                                |
+| 10:30   | [Client side validation mc](https://github.com/foundersandcoders/mc-client-side-validation)                                       | UX, usability, HTML forms                 |
+| 11:30   | [Stateless session management workshop](https://github.com/foundersandcoders/ws-jwt-stateless-session)                            | JSON Web Tokens, stateless authentication |
+| _13:00_ | _Lunch_                                                                                                                           |                                           |
+| 14:00   | [Stateless session workshop contd.](https://github.com/foundersandcoders/ws-jwt-stateless-session)                                |                                           |
+| 14:30   | Technical spikes                                                                                                                  |                                           |
+| 16:30   | Spike presentation prep                                                                                                           |                                           |
+| 17:00   | Spike presentations                                                                                                               |                                           |
+
+## Day 3
+
+| Time    | Activity                                                   | Learning outcomes   |
+| ------- | ---------------------------------------------------------- | ------------------- |
+| 09:45   | Intro reading                                              |                     |
+| 10:00   | [Promises MC](https://github.com/oliverjam/learn-promises) | Promises, fetch API |
```

Comment:
Some form of this will be given in Week 2 now, since they need to learn promises to use `fetch`. We should find a different/more relevant to the project MC for this week

Reply:
Good point. How about a new challenge where you have to refactor some bcrypt functions that hash passwords/compare a user's input with a stored hash to use promises instead of callbacks? All we'd have to do is give them the calback solution from the [day 1 password management workshop](https://github.com/foundersandcoders/ws-password-management) and get them to refactor it. What do you think?

Reply:
Could we move the Client Side Validation into this slot and then add a typography workshop to space after the design burst?

Reply:
@bobbysebolao I was planning on just teaching `bcrypt` and `pg`'s promise APIs from the start, since they'll have been doing promises the whole time.

@Albadylic I think client-side validation is a good fit actually 👍